platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/ProblemDescriptor.java

```diff
@@ -88,11 +88,10 @@ public interface ProblemDescriptor extends BaseProblemDescriptor {
     List<Solution> getSolutions();
 
     /**
-     * Returns the exception associated with this problem.
+     * Returns the failure associated with this problem.
```

Comment:
❓ Should we document what happens in the TAPI 8.7 -> Gradle 8.6 scenario (presumably `getFailure()` will always return `null`).  Do we have a test to verify that?